### PR TITLE
Update README for Chronos-2

### PR DESCRIPTION
**Pull request by abdulfatir**
*Issue #, if available:*

*Description of changes:*

TODOs:

- [x] Shorten introduction.
- [x] Change example to Chronos-2.
- [ ] ~Move pretraining related stuff to Chronos notebook.~

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,114 +96,78 @@ cd chronos-forecasting && pip install --editable ".[training]"
...
-# quantiles is an fp32 tensor with shape [batch_size, prediction_length, num_quantile_levels]
-# mean is an fp32 tensor with shape [batch_size, prediction_length]
-quantiles, mean = pipeline.predict_quantiles(
-    context=torch.tensor(df["#Passengers"]),
-    prediction_length=12,
-    quantile_levels=[0.1, 0.5, 0.9],
-)
-```
+# Load historical energy prices and past values of covariates
+context_df = pd.read_parquet("s3://autogluon/datasets/timeseries/electricity_price/train.parquet")
```
Comment (shchur) [inline README.md]:
```suggestion
context_df = pd.read_parquet("https://autogluon.s3.amazonaws.com/datasets/timeseries/electricity_price/train.parquet")
```

File: README.md [hunk 2]
```diff
@@ -102,114 +96,78 @@ cd chronos-forecasting && pip install --editable ".[training]"
...
 
-For the original Chronos models, `pipeline.predict` can be used to draw forecast samples. More options for `predict_kwargs` in `pipeline.predict_quantiles` can be found with:
+# Load future values of covariates
+test_df = pd.read_parquet("s3://autogluon/datasets/timeseries/electricity_price/test.parquet")
```
Comment (shchur) [inline README.md]:
```suggestion
test_df = pd.read_parquet("https://autogluon.s3.amazonaws.com/datasets/timeseries/electricity_price/test.parquet")
```

File: README.md [hunk 3]
```diff
@@ -45,171 +45,113 @@ For details on Chronos models, training data and procedures, and experimental re
   </span>
 </p>
```
Comment (shchur) [inline README.md]: Should we remove L36-L46?

File: README.md [hunk 4]
```diff
@@ -45,171 +45,113 @@ For details on Chronos models, training data and procedures, and experimental re
...
+ts_ground_truth.plot(ax=ax, label=f"future {target} (ground truth)", color="xkcd:grass green")
+ts_pred["predictions"].plot(ax=ax, label="forecast", color="xkcd:violet")
+ax.fill_between(
+    ts_pred.index,
+    ts_pred["0.1"],
+    ts_pred["0.9"],
+    alpha=0.7,
+    label="prediction interval",
+    color="xkcd:light lavender",
 )
```
Comment (shchur) [inline README.md]: This could be more concise:
```python
import matplotlib.pyplot as plt

ts_context = context_df.set_index(timestamp_column)[target].tail(256)
ts_pred = pred_df.set_index(timestamp_column)
ts_ground_truth = test_df.set_index(timestamp_column)[target]

ts_context.plot(label=f"historical data", color="xkcd:azure", figsize=(12, 3))
ts_ground_truth.plot(label=f"future data (ground truth)", color="xkcd:grass green")
ts_pred["predictions"].plot(label="forecast", color="xkcd:violet")
plt.fill_between(
    ts_pred.index,
    ts_pred["0.1"],
    ts_pred["0.9"],
    alpha=0.7,
    label="prediction interval",
    color="xkcd:light lavender",
)
plt.legend()
```

File: README.md [hunk 5]
```diff
@@ -20,196 +19,113 @@
 
 
 ## 🚀 News
+- **20 Oct 2025**: 🚀 [Chronos-2](https://arxiv.org/abs/2510.15821) released. It offers _zero-shot_ support for univariate, multivariate, and covariate-informed forecasting tasks. Chronos-2 achieves the best performance on fev-bench, GIFT-Eval and Chronos Benchmark II amongst pretrained models. Check out [this notebook](notebooks/chronos-2-quickstart.ipynb) to get started with Chronos-2.
```
Comment (shchur) [inline README.md]:
```suggestion
- **20 Oct 2025**: 🚀 [Chronos-2](https://arxiv.org/abs/2510.15821) released. It offers _zero-shot_ support for univariate, multivariate, and covariate-informed forecasting tasks. Chronos-2 achieves the best performance on fev-bench, GIFT-Eval and Chronos Benchmark II among pretrained models. Check out [this notebook](notebooks/chronos-2-quickstart.ipynb) to get started with Chronos-2.
```

File: README.md [hunk 6]
```diff
@@ -20,196 +19,113 @@
...
-)
+pipeline = Chronos2Pipeline.from_pretrained("s3://autogluon/chronos-2", device_map="cuda")
 
-df = pd.read_csv(
-    "https://raw.githubusercontent.com/AileenNielsen/TimeSeriesAnalysisWithPython/master/data/AirPassengers.csv"
-)
+target = "target"  # Column name containing the values to forecast
+prediction_length = 24  # Number of steps to forecast ahead
+id_column = "id"  # Column identifying different time series
+timestamp_column = "timestamp"  # Column containing datetime information
```
Comment (shchur) [inline README.md]: How about we remove these variables completely and make the example for Chronos-2 usage shorter?

```python
import pandas as pd  # requires: pip install pandas
from chronos import Chronos2Pipeline

pipeline = Chronos2Pipeline.from_pretrained("s3://autogluon/chronos-2", device_map="cuda")

# Load historical energy prices and past values of covariates
context_df = pd.read_parquet("https://autogluon.s3.amazonaws.com/datasets/timeseries/electricity_price/train.parquet")

# (Optional) Load future values of covariates
test_df = pd.read_parquet("https://autogluon.s3.amazonaws.com/datasets/timeseries/electricity_price/test.parquet")
future_df = test_df.drop(columns="target")

# Generate predictions with covariates
pred_df = pipeline.predict_df(
    context_df, 
    future_df=future_df,
    prediction_length = 24  # Number of steps to forecast
    quantile_levels=[0.1, 0.5, 0.9],  # Quantiles for probabilistic forecast
    id_column = "id"  # Column identifying different time series
    timestamp_column = "timestamp"  # Column with datetime information
    target = "target"  # Column(s) with time series values to predict
)
```
